README.md

```diff
@@ -83,5 +83,5 @@ If you want to locally test the provider binary without building from sources, y
     
 to your local using the following example: 
 
-    podman run --pull=always --rm registry.ci.openshift.org/ci/ocm-tf-e2e:latest cat /root/.terraform.d/plugins/terraform.local/local/ocm/1.0.1/linux_amd64/terraform-provider-ocm > ~/terraform-provider-ocm && chmod +x ~/terraform-provider-ocm
+    podman run --pull=always --rm registry.ci.openshift.org/ci/ocm-tf-e2e:latest cat /root/.terraform.d/plugins/terraform.local/local/ocm/1.0.4/linux_amd64/terraform-provider-ocm > ~/terraform-provider-ocm && chmod +x ~/terraform-provider-ocm
```

Comment:
if you want to make it even leaner and without the need of specifying the version:
```suggestion
    podman run --pull=always --rm registry.ci.openshift.org/ci/ocm-tf-bin:latest cat /root/terraform-provider-ocm > ~/terraform-provider-ocm && chmod +x ~/terraform-provider-ocm
```

``ocm-tf-bin`` is a new image we're publishing which is much smaller in size (219MB vs 2.18GB)